src/jvm_wrapper/jvm_instance_wrapper.h

```diff
@@ -34,6 +34,25 @@ public:
                                                                                      \
 private:
 
+#define INIT_JNI_BINDINGS_TEMPLATE(...)                                                       \
```

Comment:
The only difference I see with the regular macro is that the fqname became a property instead of a method. Is there a reason why we can't just apply this change to the regular macro instead of just having a near copy of it ?

Reply:
Yes, it is because `get_fully_qualified_name` is not accessible when using a template, it would require to call it this way: `MyTemplate<T>::get_fully_qualified_name()`, which would force to repeat class name in macro